docs/elements.md

```diff
@@ -721,6 +723,52 @@ def generate(data):
+| `weight`                   | integer                                      | 1            | Weight to use when computing a weighted average score over elements.                                                                                                                                                                                                                    |
+| `correct-answer`           | string                                       | special      | Correct answer for grading. Defaults to `data["correct_answers"][answers-name]`.                                                                                                                                                                                                        |
+| `label`                    | text                                         | &mdash;      | A prefix to display before the input box (e.g., `label="$F =$"`).                                                                                                                                                                                                                       |
+| `suffix`                   | text                                         | &mdash;      | A suffix to display after the input box (e.g., `suffix="$\rm m/s^2$"`).                                                                                                                                                                                                                 |
+| `display`                  | "block" or "inline"                          | "inline"     | How to display the input field.                                                                                                                                                                                                                                                         |
+| `grading-mode`             | "with-units", "exact-units", or "only-units" | "with-units" | How to grade student submission. "only-units" only checks for the units input by the student. "exact-units" asks for a quantity with a specified unit. "with-units" is similar to "units-fixed", but will automatically convert the units used by the given answer if possible.         |
+| `comparison`               | "exact", "sigfig", "decdig" or "relabs"      | "sigfig"     | How to grade. "relabs" uses relative ("rtol") and absolute ("atol") tolerances. "sigfig" use "digits" significant digits. "decdig" uses "digits" after decimal place, "exact" uses `==` and should only be used for integers. Attribute can only be set if `grading-mode=exact-units`   |
```

Comment:
> relabs" uses relative ("rtol") and absolute ("atol") tolerances

I have no idea what this means. I don't know if the explanation needs to be rewritten or if I am just missing some context.